service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -185,27 +188,25 @@ public void deletingAnExistingPolicy() throws Exception {
   public RequestResponsePact getPaoWithAnExistingPolicy(PactDslWithProvider builder) {
     var policyResponseShape =
         new PactDslJsonBody()
-            .valueFromProviderState("objectId", existingPolicyProviderStateValue, existingPolicyId)
+            .stringValue("objectId", existingPolicyId.toString())
             .stringMatcher("component", tpsComponentRegex)
             .stringMatcher("objectType", tpsObjectTypeRegex)
             .object("effectiveAttributes", tpsPolicyInputsObjectShape)
             .object("attributes", tpsPolicyInputsObjectShape);
 
     return builder
-        .given(existingPolicyState)
+        .given("a policy with the given id exists", Map.of("id", existingPolicyId.toString()))
         .uponReceiving("A request to retrieve a policy")
         .method("GET")
-        .pathFromProviderState(
-            "/api/policy/v1alpha1/pao/%s".formatted(existingPolicyProviderStateValue),
-            "/api/policy/v1alpha1/pao/%s".formatted(existingPolicyId))
+            .path("/api/policy/v1alpha1/pao/%s".formatted(existingPolicyId))
```

Comment:
This is the one spot where I couldn't get the existing test to pass while maintaining `pathFromProviderState`, so it's still just expecting the existingPolicyId that is being explicitly passed in by the consumer.